stable/traefik/values.yaml

```diff
@@ -26,3 +26,6 @@ dashboard:
   domain: traefik.example.com
 gzip:
   enabled: true
+kubernetes:
+  namespaces: ~
+  labelSelector: ~
```

Comment:
The effective result is  "watch all namespaces with no filter". I've commented it out and removed "~"